lib/updater.dart

```diff
@@ -136,8 +136,11 @@ Future<void> update(
 
         updatingStatus.value = status;
 
-        await github.pullRequests.list(slug, pages: 1000).forEach(
-            (pr) async => await addPullRequestToDatabase(prRef, pr, logger));
+        await github.pullRequests.list(slug, pages: 1000).forEach((pr) async {
+          final list = await getReviewers(github, slug, pr);
+          pr.requestedReviewers?.addAll(list);
```

Comment:
Will this leave us with some duplicate `User` entries in the list? Perhaps de-dup as we add people.

Reply:
And thinking about this, it would be nice to know that `requestedReviewers` is generally empty (and mutable) instead of being null when there are no requested reviewers.

Reply:
If we want to avoid creating more model objects here, you could effectively create a new field on the PullRequest class via using an expando to store into a 'reviewers' field and using an extension method to pull the info out again.